bazel/grpc_deps.bzl

```diff
@@ -316,15 +316,15 @@ def grpc_deps():
             sha256 = "122ebf7fe7d1c8e938af6aeaee0efe788a3a2449ece5a8d6a428cb18d6f88033",
         )
 
-    if "libuv" not in native.existing_rules():
+    if "com_github_libuv_libuv" not in native.existing_rules():
         http_archive(
-            name = "libuv",
+            name = "com_github_libuv_libuv",
             build_file = "@com_github_grpc_grpc//third_party:libuv.BUILD",
-            sha256 = "dfb4fe1ff0b47340978490a14bf253475159ecfcbad46ab2a350c78f9ce3360f",
-            strip_prefix = "libuv-15ae750151ac9341e5945eb38f8982d59fb99201",
+            sha256 = "47dd6df502d09f6bffd6b12ce776d5f0033fc41de02c32f85ae96ea7f74ff560",
+            strip_prefix = "libuv-e45f1ec38db882f8dc17b51f51a6684027034609",
```

Comment:
nit: add a note that the commit corresponds to https://github.com/libuv/libuv/releases/tag/v1.35.0?